heliosbooth/templates/seal.html

```diff
@@ -29,7 +29,7 @@ <h3>Review your Ballot</h3>
 <div style="margin-left: 15px;">&#x2713; {$T.choice}</div>
 {#/for}
 {#if $T.choices[$T.question$index].length < $T.question.max}
-[you under-voted: you may select up to {$T.question.max}]
+[you selected {$T.choices[$T.question$index].length} candidates; you may select from zero to {$T.question.max}]
```

Comment:
it's not necessarily zero, that should probably be:

```suggestion
[you selected {$T.choices[$T.question$index].length} candidates; you may select from {$T.question.min} to {$T.question.max}]
```

Reply:
That is better with $T.question.min.  Did you adjust that before merge?